### PR TITLE
Data model changes to associate a project with a funding application

### PR DESCRIPTION
**Pull request by stuartmccoll**
## Proposed changes

This pull request contains changes to the funding-frontend data model which associate a `project` with a `funding_application` for extensibility purposes.

## Further comments

N/A.

## Screenshot (if applicable)

N/A.

## Checklist

- [X] Added tests for new functionality, or updated existing tests
- [X] Updated `README.md` if necessary

### CODE REVIEW

**Review by stuartmccoll — COMMENTED** [review]
@ptrelease I've been through this once and made a few amendments. It looks fine to me - tests are passing. I've already done an end-to-end manual test, but I'd like to do another before we merge it in.

**Comment by stuartmccoll** [comment]
Also, @ptrelease - I think you'll need to remove the `.DS_Store` file on your end and push that up as a change - then make sure it's in your global `.gitignore` file.

File: app/views/funding_application/gp_project/check_answers/_heritage.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<div class="govuk-summary-list__row">
+  <dt class="govuk-summary-list__key">
+    <%= t('views.project.check_your_answers.heritage') %>
+  </dt>
+  <dd class="govuk-summary-list__value">
+    <%= @funding_application.project.heritage_description.present? ? @funding_application.project.heritage_description : t('views.project.check_your_answers.not_answered')%>
```
Comment (ptrelease) [inline app/views/funding_application/gp_project/check_answers/_heritage.html.erb:6]: correct t

File: app/views/funding_application/gp_project/check_answers/_location.html.erb [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<div class="govuk-summary-list__row">
+  <dt class="govuk-summary-list__key">
+    <%= t('views.project.check_your_answers.where_is_your_project_taking_place') %>
```
Comment (ptrelease) [inline app/views/funding_application/gp_project/check_answers/_location.html.erb:3]: correct t

File: app/views/dashboard/show.html.erb [hunk 3]
```diff
@@ -65,59 +65,37 @@
...
-    <% end %>
-  </ul>
-
-  <% end %>
-
   <% if @funding_applications.present? %>
 
     <h3 class="govuk-heading-m">
-      <%= "#{I18n.t("funding_programmes.covid_19_recovery_loan")} #{I18n.t("dashboard.applications")}" %>
+      <%= "#{I18n.t("funding_programmes.open_small_three_to_ten_k")} #{I18n.t("dashboard.applications")}" %>
```
Comment (ptrelease) [inline app/views/dashboard/show.html.erb:71]: check t

Comment (stuartmccoll) [reply app/views/dashboard/show.html.erb:71]: This is correct 👍.

**Review by ptrelease — APPROVED** [review]

**Pull request merged**